.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "sorc/post_gtg.fd"]
```

Comment:
@aerorahul This line might be changed as "sorc/ncep_post.fd/post_gtg.fd"

Reply:
I think 'sorc/post_gtg.fd' right after 'submodule' is just a name, while the actual path matters. I am able to checkout GTG code with:
git -c submodule."sorc/post_gtg.fd".update=checkout submodule update --init --recursive

To prove what I am saying, I am also able to checkout GTG code after the following modification:
modified the checkout command line to:
git -c submodule."just_a_name".update=checkout submodule update --init --recursive
and modified .gitmodules as:
[submodule "just_a_name"]
	path = sorc/ncep_post.fd/post_gtg.fd
	url = https://github.com/NCAR/UPP_GTG
        update = none

But Wen made a good point to avoid a name which might cause confusing.

Reply:
Done in da1a313

Reply:
@aerorahul Thank you. @WenMeng-NOAA with the newest update, please remember to update the wiki page of GTG Access. The checkout command line will be:
git -c submodule."post_gtg.fd".update=checkout submodule update --init --recursive